examples/vector_databases/Vector_db_introduction.ipynb

```diff
@@ -0,0 +1,1187 @@
+      "17. Title: Central Europe Certainty: 0.9212862849235535\n",
+      "18. Title: Art Certainty: 0.9208334386348724\n",
+      "19. Title: European Certainty: 0.92069211602211\n",
+      "20. Title: Byzantine art Certainty: 0.920437216758728\n"
+     ]
+    }
+   ],
+   "source": [
+    "query_result = query_weaviate('modern art in Europe','Article')\n",
+    "counter = 0\n",
+    "for article in query_result['data']['Get']['Article']:\n",
+    "    counter += 1\n",
+    "    print(f\"{counter}. Title: {article['title']} Certainty: {article['_additional']['certainty']}\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 44,
+   "id": "93c4a696",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "1. Title: Historic Scotland Certainty: 0.9464837908744812\n",
+      "2. Title: First War of Scottish Independence Certainty: 0.9461104869842529\n",
+      "3. Title: Battle of Bannockburn Certainty: 0.9455609619617462\n",
+      "4. Title: Wars of Scottish Independence Certainty: 0.944368839263916\n",
+      "5. Title: Second War of Scottish Independence Certainty: 0.9395008385181427\n",
+      "6. Title: List of Scottish monarchs Certainty: 0.9366503059864044\n",
+      "7. Title: Kingdom of Scotland Certainty: 0.935274213552475\n",
+      "8. Title: Scottish Borders Certainty: 0.9317866265773773\n",
+      "9. Title: List of rivers of Scotland Certainty: 0.9296278059482574\n",
+      "10. Title: Braveheart Certainty: 0.9294214248657227\n",
+      "11. Title: John of Scotland Certainty: 0.9292325675487518\n",
+      "12. Title: Duncan II of Scotland Certainty: 0.9291643798351288\n",
+      "13. Title: Bannockburn Certainty: 0.9291241466999054\n",
+      "14. Title: The Scotsman Certainty: 0.9280610680580139\n",
+      "15. Title: Flag of Scotland Certainty: 0.9270428121089935\n",
+      "16. Title: Banff and Macduff Certainty: 0.9267247915267944\n",
+      "17. Title: Guardians of Scotland Certainty: 0.9260919094085693\n",
+      "18. Title: Scottish Parliament Certainty: 0.9252097904682159\n",
+      "19. Title: Holyrood Abbey Certainty: 0.925055593252182\n",
+      "20. Title: Scottish Certainty: 0.9249534606933594\n"
+     ]
+    }
+   ],
+   "source": [
+    "query_result = query_weaviate('Famous battles in Scottish history','Article')\n",
+    "counter = 0\n",
+    "for article in query_result['data']['Get']['Article']:\n",
+    "    counter += 1\n",
+    "    print(f\"{counter}. Title: {article['title']} Certainty: {article['_additional']['certainty']}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ad74202e",
+   "metadata": {},
+   "source": [
+    "Thanks for following along, you're now equipped to set up your own vector databases and use embeddings to do all kinds of cool things - enjoy! For more complex use cases please continue to work through the cookbook examples here:\n",
+    "\n",
+    "TODO: Make other cool things to link to"
```

Comment:
Still TODO I suppose

Reply:
I've stripped out this TODO, will update it with references once we have more material to link to